src/ray/protobuf/gcs.proto

```diff
@@ -703,7 +703,8 @@ message JobTableData {
   // The optional JobInfo from the Ray Job API.
   optional JobsAPIInfo job_info = 10;
   // Whether this job has running tasks.
-  bool is_running_tasks = 11;
+  // In GetAllJobInfo, if GCS can't reach the driver, it will be unset.
+  optional bool is_running_tasks = 11;
```

Comment:
why we add `optional`?

Reply:
when GCS handles `JobInfoGcsService.GetAllJobInfo` it sends 1 RPC to each driver for each job's number of running tasks, to determine this `is_running_tasks`. If the RPC times out (1s), GCS has no idea if it's running or not (the job may be hang, unresponsive, connection issues and whatsoever). We want GCS to share this knowledge to the caller and hence make it optional.